docs/reference/config-yml-reference.md

```diff
@@ -34,6 +34,21 @@ according to the rules documented on [the file structure](../file-structure.md#d
 
 :::
 
+## Schema Validation
+
+A schema file is provided for `settings.yml` to help assist in editing the file. To use it, simply
+add the below snippet to the first line in your `settings.yml` file:
```

Comment:
If that is the case, shouldn't the entire schema be put into it's own page?

Reply:
I'm not sure what you mean. There's more than one schema.

Reply:
I realized there are some words missing.  I was meaning to say, shouldn't there be a whole page related to the schema's.  Instead of having it noted for each one in each individual section, it has it's own section so if there is ever additional schema's defined we only have to update one section.

Reply:
When I originally envisioned this documentation, I thought of it as its own page, but since you're doing the work I was going to be OK with how you had it. If you think there's a better way to organize it, I'm happy to review that too.

Reply:
I'll add it to the back burner.  This works for now, as I need to figure out the scope of the doc and clear it with you before I commit time on it.